### ISSUE TITLE
NTR: CT or MRI: paravertebral mass (Lab: radiology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8987]
New term request CT or MRI: paravertebral mass (elthat_130212191336): Paraortic sympathetic chain; paraganglia

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000660
- label: Paravertebral mass
- parents: [Abnormality of the vertebral column (HP:0000925)](https://hpo.jax.org/app/browse/term/HP:0000925)
- pmid: PMID:30517335
- def: A swelling or enlargment localized next to the vertebral column. The word mass is usually used at an early stage of the diagnostic workup before the precise nature of the swelling has been determined.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8987
- synonyms:
